### PR TITLE
Use releases v2 API instead of revision history

### LINKED ISSUE #968: Move to the new v2 API endpoint

### PR DESCRIPTION
**Pull request by bartaz**
Fixes #968 

### QA
- ./run or demo
- Go to releases page of some snaps
- Everything should work as before
- You can compare with public view to see that same revisions are shown as released

Also test locally with RELEASE_UI_ENABLED flag turned off. You may compare with public view

### CODE REVIEW

File: webapp/api/dashboard.py [hunk 1]
```diff
@@ -71,6 +71,17 @@
...
 
 
+DASHBOARD_API_V2 = os.getenv(
+    'SNAPCRAFT_IO_API_V2',
+    'https://dashboard.snapcraft.io/api/v2/'
+)
+
+SNAP_RELEASE_HISTORY_URL = ''.join([
+    DASHBOARD_API_V2,
+    'snaps/{snap_name}/releases?page=1&size=100'
```
Comment (bartaz) [inline webapp/api/dashboard.py]: By default page size is 500, but it seems that it results in time out for some bigger snaps (node, android-studio). Reducing size to 100 seems to help for some, but others still time out (node).

I'm not sure if it's an issue on our side (should we wait longer) or on store side.

Comment (Lukewh) [reply webapp/api/dashboard.py]: I think it's probably our side. We're pretty tight on the timeouts

File: webapp/api/dashboard.py [hunk 2]
```diff
@@ -71,6 +71,17 @@
 ])
 
 
+DASHBOARD_API_V2 = os.getenv(
+    'SNAPCRAFT_IO_API_V2',
```
Comment (tbille) [inline webapp/api/dashboard.py]: 'DASHBOARD_API_V2'

Comment (bartaz) [reply webapp/api/dashboard.py]: Ah, good catch - I renamed the var in code but not in env.

**Pull request merged**